bedrock/firefox/templates/firefox/all.html

```diff
@@ -174,12 +181,12 @@ <h4 class="not-found{% if builds %} hide{% endif %}">{{ _('No matching languages
 
 <!--
   build: Locale dictionary from product details.
-  platform: Value in build.platforms to use. One of 'Windows', 'OS X', 'Linux'.
+  platform: Value in build.platforms to use. One of 'Windows', 'Windows 64', 'OS X', 'Linux' and 'Linux 64'.
   tooltip: Text to display as tooltip for download link.
 -->
 {% macro build_link(build, platform, tooltip) %}
   {% if build.platforms[platform] %}
-    <td class="download {% if platform == 'Windows' %}win{% elif platform == 'OS X' %}mac{% elif platform == 'Linux' %}linux{% elif platform == 'Linux 64' %}linux64{% endif %}"><a href="{{ build.platforms[platform].download_url }}" title="{{ tooltip }}">{{ _('Download') }}</a></td>
+    <td class="download {{ platform|lower|replace(' ', '') }}"><a href="{{ build.platforms[platform].download_url }}" title="{{ tooltip }}">{{ _('Download') }}</a></td>
```

Comment:
I have replaced `mac` with `osx` in `all.less` so it should be safe. `win` is not used here.